src/plugins/fcrypt/testmod_fcrypt.c

```diff
@@ -114,6 +114,28 @@ static int isTestFileCorrect (const char * file)
 	return returnValue;
 }
 
+static int gpg_available (void)
+{
+	int available = 0;
```

Comment:
Please avoid this code duplication if possible.

Reply:
Is there a module for common test cases? Otherwise it will be complicated to avoid the duplication.

Reply:
@petermax2 cmake has find_program https://cmake.org/cmake/help/v3.0/command/find_program.html

pass the result into your test files and `#if` your gpg tests.

afterwards you can remove the function completely